README.md

```diff
@@ -1,26 +1,178 @@
 # VimeoNetworking
-The Vimeo iOS SDK
 
-##### NOTE: This library is under active and rapid development. Code changes, an official release, and documentation are all to follow, stay tuned!
+**VimeoNetworking** is the authoritative Swift networking library for the Vimeo API.  Fully designed and implemented with Swift in mind, **VimeoNetworking** is type-safe, well `enum`erated, and never, ever, *ever* force-unwrapped. 
 
-## Found an Issue?
+##### Hey Creator, if you're primarily interested in uploading videos to Vimeo, you should also check out [VimeoUpload](https://github.com/vimeo/VimeoUpload).
 
-Please file it in the git [issue tracker](https://github.com/vimeo/VimeoNetworking/issues).
+## Installing with CocoaPods
 
-## Want to Contribute?
+To get started integrating `VimeoNetworking`, add the following lines to your `Podfile` and run `pod install`:
 
-If you'd like to contribute, please follow our guidelines found in [CONTRIBUTING.md](CONTRIBUTING.md).
+```Ruby
+use_frameworks! # required for Swift frameworks
 
-## License
+target 'YourAppTargetName' do
+	pod 'VimeoNetworking', '1.0'
+end
+```
 
-`VimeoNetworking` is available under the MIT license. See the [LICENSE](LICENSE) file for more info.
+## Initialization
 
-## Questions?
+The first step towards using the Vimeo API is registering a new application on the Vimeo Developer site: [My API Apps](https://developer.vimeo.com/apps).  You'll need a Vimeo account, if you don't already have one.
 
-Tweet at us here: [@vimeoapi](https://twitter.com/vimeoapi).
+### App Configuration
 
-Post on [Stackoverflow](http://stackoverflow.com/questions/tagged/vimeo-ios) with the tag `vimeo-ios`.
+Once you have a new app setup, click into its authentication settings and make note of the "Client Identifier" and "Client Secret" fields.  Next, determine which `Scope` permissions your application requires from the available options listed here: [Supported Scopes](https://developer.vimeo.com/api/authentication#scopes).  Use this information to instantiate a new `AppConfiguration` value:
 
-Get in touch [here](https://vimeo.com/help/contact).
+```Swift
+let appConfiguration = AppConfiguration(
+		clientIdentifier: "Your client identifier goes here",
+		clientSecret: "Your client secret goes here",
+		scope: [.Public, .Private, .Interact])
```

Comment:
Add a comment after the scope line saying something like, "replace with your scopes"?